src/settings.ts

```diff
@@ -428,5 +430,14 @@ export default {
 
   importer: {
     analyse: localstorageprop('importer.analyse', false)
+  },
+
+  training: {
+    puzzleBufferLen: 50,
+    ratingDiffThreshold: 100,
+    unsolvedPuzzles: localstorageprop<OfflinePuzzle[]>('training.unsolvedPuzzles', []),
```

Comment:
I'd put the last 4 props in a `training/state.ts` file instead, since it's not really settings but state.